src/component/TodoApp/TodoApp.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoInfo } from '../TodoInfo/TodoInfo';
+
+interface TodoAppProps {
+  todos: Todo[];
+  deleteTodo: (id: number) => void;
+  tempTodo: Todo | null;
+  updateTodoCompleted: (id: number, completed: boolean) => void;
+}
+
+export const TodoApp: React.FC<TodoAppProps> = ({
+  todos,
+  deleteTodo,
+  tempTodo,
+  updateTodoCompleted,
+}) => {
+  return (
+    <section className="todoapp__main">
+      {todos.map((todo) => (
+        <TodoInfo
+          todo={todo}
+          key={todo.id}
+          deleteTodo={deleteTodo}
+          updateTodoCompleted={updateTodoCompleted}
+        />
+      ))}
+      {tempTodo && (
```

Comment:
As mentioned above, the 'tempTodo' prop is not necessary. Remove this section of code.